.gitignore

```diff
@@ -11,3 +11,10 @@ docs/_build
 regparser.egg-info/
 
 .eregs_index/
+
+# eregs output
+/fr_cache.sqlite
+/diff/
+/layer/
+/notice/
+/regulation/
```

Comment:
https://github.com/18F/regulations-parser#using-build_from:
 "Running the command will generate four folders, regulation, notice, layer and possibly diff in the OUTPUT_DIR (current directory by default)."